.github/workflows/wheels.yml

```diff
@@ -15,7 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        # macOS-11 has no Python 3.6.
+        os: [ubuntu-latest, windows-latest, macOS-10.15]
```

Comment:
Wait, what? cibuildwheel does not use the Python versions on the host, so why does that matter?

Reply:
On macOS, it's not necessary to build on old platforms, the latest is fine/best. Apple uses MACOS_DEPLOYMENT_TARGET to control the target platform, and they require every runtime feature to be correctly gated (one benefit of being a company rather than random contributors); newer versions are therefore better, since you can use the newer compile time features (like header only parts of C++17), and it will target the correct runtime (which is much less than 10.15 usually, 10.9 by default or you can set 10.12 or so). If it can't, it will error out correctly.

GitHub does remove older images eventually, windows-2016 gets the ax in March. But it's fine, as we should have full support for macos-11.

Reply:
(It's probably perfectly safe to be on 10.15 for quite some time, but just noting this in response to the commit message)

Reply:
ah good to know! i remember dimly that pyca always built with old macOSes

Reply:
Using the correct (python.org) Python + MACOSX_DEPLOYMENT_TARGET is the correct solution. I know some packages try to build on old macOS (10.14 is the earliest offered by GHA) and then just required 10.14+ for users, but cibuildwheel is careful to do this correctly.

Reply:
Removed: https://github.com/hynek/argon2-cffi-bindings/commit/e8c10163ebb1ca3d7362e46cbd321dd561608d5f